### DISCUSSION TITLE
Support for update feature on monorepo or local template

### DISCUSSION THREAD

**Discussion by hevandro-veiga** [copier-org/copier#1744] [Q&A]
Hello! First, let me say that Copier is awesome, and I love it!
I’m trying to use Copier for a new project, but our team has adopted a monorepo architecture, and I need to place the template and the generated project in the same repo, but in different folders.
For example:

```
monorepo
├── apps
│   └── python
│        └── templates
│            └── awesome-copier-template
│             ...
└── only-awesome-generated-projects
    └── new-awesome-project
        ...
```

It would be great if Copier could support this kind of architecture, so I could update the template and the generated project within the same repository.

**Comment by yajo** [comment]
IIRC the destination can be any sub-folder of a repo.

However, the source must be a repo because otherwise there's no way to support migrations.

Well, we *could* support them, but the versioning would be for the whole monorepo, which usually doesn't make much sense. And in any case, support is not included right now.

So, your templates must be individual repos as a technical requirement.
